anchore_engine/configuration/localconfig.py

```diff
@@ -112,6 +115,16 @@ def analyzer_paths():
     return ANALYZER_SEARCH_PATHS
 
 
+def register_policy_bundle_source_dir(source_dir):
+    global POLICY_BUNDLE_SOURCE_DIRS
+    POLICY_BUNDLE_SOURCE_DIRS.append(source_dir)
+    load_policy_bundle_paths()
```

Comment:
I'd like to refactor this to not have this extra call here, but as the config/bootstrapping currently exists this is necessary because in https://github.com/anchore/anchore-engine/blob/e4d21c6f8171027e03b974ed8a0d570dcc20e190/anchore_engine/twisted.py#L217 when the service starts up, the path that leads to `load_policy_bundle_paths()` below is called before the service is initialized, so if the service adds policy bundles to the config object via a handler it will be to late, even if with a pre_config handler.

This is maybe slightly hacky, but multiple calls to load_policy_bundle_paths() will have no effect on existing bundles in the case where subsequent calls are just adding more bundles.